addons/binding/org.openhab.binding.russound/README.md

```diff
@@ -36,144 +49,146 @@ The following configurations occur for each of the bridges/things:
-| lasterror          | R          | String       | The last error that occurred in the zone                             |
-| enabled            | R          | Switch       | Whether the zone is enabled or not                                   |
-| repeat             | W          | Switch       | Toggle the repeat mode for the current source                        |
-| shuffle            | W          | Switch       | Toggle the shuffle mode for the current source                       |
-| rating             | W          | Switch       | Signal a like (ON) or dislike (OFF) to the current source            |
-| keypress           | W          | String       | (Advanced) Send a keypress from the zone                             |
-| keyrelease         | W          | String       | (Advanced) Send a keyrelease from the zone                           |
-| keyhold            | W          | String       | (Advanced) Send a keyhold from the zone                              |
-| keycode            | W          | String       | (Advanced) Send a keycode from the zone                              |
-| event              | W          | String       | (Advanced) Send an event from the zone                               |
-| systemfavorites    | RW         | String*      | The JSON representation for system favorites                         |
-| zonefavorites      | RW         | String**     | The JSON representation for zone favorites                           |
-| presets            | RW         | String***    | The JSON representation for zone presets                             |
-| mminit             | W          | Switch****   | Whether to initial a media management session (ON) or close an existing one (OFF) |
-| mmcontextmenu      | W          | Switch****   | Whether to initial a media management context session (ON) or close an existing one (OFF) |
+
+| Channel Type ID    | Read/Write | Item Type  | Description                                                                               |
+|--------------------|------------|------------|-------------------------------------------------------------------------------------------|
+| name               | R          | String     | The name of the zone (changed by SCS-C5 software)                                         |
+| source             | RW         | Number     | The (physical) number for the current source                                              |
+| bass               | RW         | Number     | The bass setting (-10 to 10)                                                              |
+| treble             | RW         | Number     | The treble setting (-10 to 10)                                                            |
+| balance            | RW         | Number     | The balance setting (-10 &lsqb;full left&rsqb; to 10 &lsqb;full right&rsqb;)                                  |
```

Comment:
I see an exception in line width here, in which order did you do the fixes?

Reply:
@martinvw maybe you can help me understand something.  I was getting a linter error caused by left and right square brackets.  It seems that the linter thinks these are references to a note somewhere else in the article as in:

```java
Wikipedia[1] is a great site!

[1] https://www.wikipedia.org
```
So I escaped the left and right brackets with `&lsqb;` and `&rsqb;`  So the one line of the table is now longer than all the others.  I assumed that when it is rendered by Jekyll, the substitutions would be made and the line would then be the correct length.  But perhaps I am wrong? 
WDYT?

Reply:
I assume that we do the formatting to make the text variant look great, so just treat it as 5 chars.

Reply:
@martinvw I am not clear.  Should I take the `&lsqb` out and put a "`[`" in?

Reply:
No `&lsqb` is fine but just treat it as the width it has, I assume that is the default behaviour if your reformat it after having replaces them by `&lsqb`